gradle/libs.versions.toml

```diff
@@ -76,6 +76,7 @@ junit-bom = { module = "org.junit:junit-bom", version = "5.11.4" }
 logback-classic = { module = "ch.qos.logback:logback-classic", version = "1.5.16" }
 micrometer-bom = { module = "io.micrometer:micrometer-bom", version = "1.14.3" }
 mockito-core = { module = "org.mockito:mockito-core", version = "5.15.2" }
+mockito-inline = { module = "org.mockito:mockito-inline", version = "5.2.0" }
```

Comment:
Hm. This one looks quite unmaintained - last release was 2023. Do we really need it?